tests/integration/test_exodus.py

```diff
@@ -98,3 +98,19 @@ def test_content_type_header_HEAD(cdn_test_url, testdata_path):
     r = requests.head(url)
     assert r.status_code == 200
     assert_content_type(url, r.headers["content-type"])
+
+
+testdata_origin_alias_path = [
+    "/origin/rpms/bash/4.4.19/8.el8_0/fd431d51/bash-4.4.19-8.el8_0.x86_64.rpm",
+    "/origin/rpm/bash/4.4.19/8.el8_0/fd431d51/bash-4.4.19-8.el8_0.x86_64.rpm",
+    "/content/origin/rpms/bash/4.4.19/8.el8_0/fd431d51/bash-4.4.19-8.el8_0.x86_64.rpm",
+    "/content/origin/rpm/bash/4.4.19/8.el8_0/fd431d51/bash-4.4.19-8.el8_0.x86_64.rpm",
+]
+
+
+@pytest.mark.parametrize("testdata_path", testdata_origin_alias_path)
+def test_origin_path_alias(cdn_test_url, testdata_path):
+    url = cdn_test_url + testdata_path
+    r = requests.head(url)
+    assert r.status_code == 200
+    assert r.headers["etag"] == '"810acb1a99257c23d29b76ca1de01da3"'
```

Comment:
Can you please not depend on ETag in this way?  We aren't setting any guarantees on the particular values used by ETag, and according to the S3 docs, you can also end up with different etags if you use multipart upload - which we may well end up doing.

Other suggestions for what to do here:

- use Want-Digest/Digest (I think you did this already but decided you didn't like it since it means the test is now depending on multiple features.  That concern makes sense, but I think we could live with it...)
- or actually download the RPMs and make sure they're all equal,
- or use ETag, but instead of hardcoding the expected value, fetch all 4 paths and just verify that they all have the same ETag without specifying the value

Reply:
@rohanpm I will go with "Want-Digest/Digest", because the test logic will be clean and simple.